### PR TITLE
SYSGOOD_FLAG constant

### PR DESCRIPTION
**Pull request by spoutn1k**
Removal of hardcoded constant to fix fringe error.

### CODE REVIEW

File: ptrace/debugger/syscall_state.py [hunk 1]
```diff
@@ -37,7 +38,10 @@ def exit(self):
                 and not self.process.debugger.trace_exec:
             # Ignore the SIGTRAP after exec() syscall exit
             self.process.syscall()
-            self.process.waitSignals(SIGTRAP, SIGTRAP | 0x80)
+            signum = SIGTRAP
+            if self.process.debugger.use_sysgood:
+                signum |= SYSGOOD_FLAG
```
Comment (vstinner) [inline ptrace/debugger/syscall_state.py:43]: ah, that sounds better! it's correct according to https://man7.org/linux/man-pages/man2/ptrace.2.html

File: ptrace/debugger/syscall_state.py [hunk 2]
```diff
@@ -37,7 +38,10 @@ def exit(self):
...
+            self.process.waitSignals(signum)
```
Comment (vstinner) [inline ptrace/debugger/syscall_state.py:44]: Using waitSyscall() here doesn't work?

**Comment by spoutn1k** [comment]
Actually, testing further, this does not work either for `bash -c "ls"` either, while it works for the forking example I shared in #66. The unhandled exception is the SIGTRAP signal, that apparently is not flagged with the SYSGOOD mask.

@vstinner would oyu happen to be able to replicate #66 / have an idea of what happens in this case ? When stracing a forking program without options a stray SIGTRAP | SYSGOOD is caught, resulting in an exception. The only method I found to fix it was using what @neok-m4700 pushed earlier. This pull request does the exact same thing as [`68833a`](https://github.com/vstinner/python-ptrace/commit/68833a9e6fd9ee6353a3a1c9c7a65b2fe300991f) and it is as broken.

**Comment by vstinner** [comment]
I'm not longer using python-ptrace, so I don't want to spend too much time on maintaining it.

> The unhandled exception is the SIGTRAP signal, that apparently is not flagged with the SYSGOOD mask.

The Linux ptrace API is badly documented. I mostly wrote python-ptrace with trial and error, treating the Linux kernel as a black box.

**Pull request closed without merging**